src/styles/searchbar.css

```diff
@@ -22,6 +23,12 @@
   top: 0;
   background: url('../assets/icons/16/icon-magnifier-16.svg') .6em center no-repeat;
 }
+[role=banner] .coz-searchbar-autosuggest-container.--searching:before{
+  padding-left: .5em;
+  background: url('../assets/icons/spinner.svg') center center no-repeat;
```

Comment:
You could have write instead…
```
background: url('../assets/icons/spinner.svg') center center / 1em no-repeat;
```
… but I'm just being meticulous 😛

Reply:
You're, right, stupid copy pasting.